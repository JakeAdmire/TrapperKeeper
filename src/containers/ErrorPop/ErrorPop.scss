.error-pop-container {
  background: #DD2D00;
  color: white;
  h1 {
    font-size: 1rem;
  }
}