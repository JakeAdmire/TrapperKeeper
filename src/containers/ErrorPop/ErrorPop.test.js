import React from 'react';
import { ErrorPop, mapStateToProps } from './ErrorPop';
import { shallow } from 'enzyme';

describe('ErrorPop', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <ErrorPop/>
    )
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });

  describe('mapStateToProps', () => {

    it('should return a string', () => {
      const mockState = {
        error: 'Title is Required',
        allNotes: [
          { id: "123ABA", 
            title: 'Worf ToDo', 
            list: [
              { id: "123", text: 'Eat food', isComplete: false },
            ]
          },
          { id: "987GDGFD",
            title: 'Jake ToDo',
            list: [
              { id: "0098", text: 'Do styling', isComplete: false },
            ]
          }
        ]
      }; 
      const expected = {error: 'Title is Required'};

      const mappedProps = mapStateToProps(mockState);
      expect(mappedProps).toEqual(expected);
    });
    
  });
});
