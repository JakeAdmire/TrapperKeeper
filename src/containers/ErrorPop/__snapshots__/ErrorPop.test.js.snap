// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPop should match the snapshot 1`] = `
<div
  className="error-pop-container"
>
  <h1 />
</div>
`;
